apps/src/block_utils.js

```diff
@@ -1063,67 +1063,18 @@ exports.createJsWrapperBlockCreator = function (
           this.setPreviousStatement(true);
         }
 
+        // Boolean constant to store when we show mini-toolbox.
         // Use window.appOptions, not global appOptions, because the levelbuilder
         // block page doesn't have appOptions, but we *do* want to show the mini-toolbox
-        // there
-        if (
+        // there.
+        const showMiniToolbox =
```

Comment:
Nice idea. This will help to simplify logic for Google Blockly fields that need to be created later on.